config/routes.rb

```diff
@@ -100,6 +100,16 @@
       resources :calculator_constants, only: :index, defaults: { format: :json }
     end
 
+    scope :preneeds do
+      resources :cemeteries, only: :index, defaults: { format: :json }
+      resources :preneeds_states, only: :index, defaults: { format: :json }
+      resources :preneeds_attachment_types, only: :index, defaults: { format: :json }
+      resources :discharge_types, only: :index, defaults: { format: :json }
+      resources :military_ranks, only: :index, defaults: { format: :json }
+      resources :branches_of_service, only: :index, defaults: { format: :json }
+      resources :applications, only: :create, defaults: { format: :json }
+    end
```

Comment:
@saneshark This is actually "namespaces" in the code (checked the preneeds-initial branch on github) - I am not sure why it is showing as scoped here...